go.mod

```diff
@@ -1,23 +1,208 @@
 module github.com/transcend-io/terragrunt-atlantis-config
 
-go 1.16
+go 1.21
+
+toolchain go1.21.3
```

Comment:
I'm ready to drop off the idea of bumping the Go version here - it should simplify the diff on this file. WDYT?

Reply:
it looks much like dependencies update, I think we're good with that